docs/source/_toctree.yml

```diff
@@ -1,6 +1,10 @@
 - sections:
   - local: index
     title: 🤗 Hugging Face Hub Python Wrapper
+  - local: quick-start
+    title: Quick start
+  title: "Get started"
```

Comment:
nit: there are two titles here

Reply:
I think it makes sense to have two different titles: `Get started` is the more general section title and `Quick start` is the title for a specific page :)

Reply:
Ah yes, I didn't see the whole file format. Thanks!